### PR TITLE
[AVAX] Merge v1.7.0

### PR DESCRIPTION
**Pull request by evlekht**
## Why this should be merged
This PR contains changes from ava-labs/avalanche-network-runner between 1.4.0 and 1.7.0.
https://github.com/ava-labs/avalanche-network-runner/releases/tag/v1.7.0
https://github.com/ava-labs/avalanche-network-runner/releases/tag/v1.6.0
https://github.com/ava-labs/avalanche-network-runner/releases/tag/v1.5.0
https://github.com/ava-labs/avalanche-network-runner/releases/tag/v1.4.1
### Conflicts
- .github/workflows/release.yml
- README.md
- go.mod
- go.sum
- local/network.go
- scripts/tests.e2e.sh
- tests/e2e/e2e_test.go
- utils/utils.go
## How this was tested
With already existing unit-tests, e2e tests and caminogo e2e tests.

### CODE REVIEW

File: local/deprecatedFlagsSupport.json [hunk 1]
```diff
@@ -10,5 +10,11 @@
       "old_name": "whitelisted-subnets",
       "new_name": "track-subnets",
       "value_map": ""
+  },
+  {
+      "version": "v1.10.4",
```
Comment (knikos) [inline local/deprecatedFlagsSupport.json]: we have to adjust this version to caminogo's version ie v1.1.4-rc0. Not sure if we can reference RCs here though

File: scripts/tests.e2e.sh [hunk 2]
```diff
@@ -25,6 +25,40 @@ TEMP_PATH=/tmp
 mkdir $TEMP_PATH/camino-node-1 -p
 cp -r $CAMINO_NODE_PATH $TEMP_PATH/camino-node-1/camino-node
 
+DEFAULT_SUBNET_EVM_VERSION=0.5.1
+SUBNET_EVM_VERSION=$DEFAULT_SUBNET_EVM_VERSION
+
+echo "Running e2e tests with:"
+echo VERSION_1: ${VERSION_1}
+echo VERSION_2: ${VERSION_2}
```
Comment (knikos) [inline scripts/tests.e2e.sh]: was these 2 intentionally re-introduced?

**Review by knikos — CHANGES_REQUESTED** [review]

File: scripts/tests.e2e.sh (hunk 2 above)
Comment (evlekht) [reply scripts/tests.e2e.sh]: No, thanks for the catch

File: local/deprecatedFlagsSupport.json (hunk 1 above)
Comment (evlekht) [reply local/deprecatedFlagsSupport.json]: CNR will get version from node, which will be `"caminogo: v1.1.4-rc0, commit: 3e2156c73\ncompat: v1.1.5 [database: v1.4.5]\n"`
Then it cuts it down to `"v1.1.4"` and compare it with value from json

Comment (evlekht) [reply local/deprecatedFlagsSupport.json]: But jup, thanks, should replace 1.10.4 with 1.1.4

**Review by knikos — APPROVED** [review]

**Pull request merged**